setup.py

```diff
@@ -52,6 +52,7 @@
         "click>=7.0,<8.0",
         "watchdog>=0.10.0",
         "gunicorn>=19.2.0,<21.0; platform_system!='Windows'",
+        "cloudevents==1.0.1",
```

Comment:
```suggestion
        "cloudevents>=1.0,<2.0",
```
We should never explicitly pin a sub-dependency of a library.